.travis.yml

```diff
@@ -107,6 +107,9 @@ jobs:
           done
         - source "$HOME/miniconda/etc/profile.d/conda.sh"
         - export PATH="$HOME/miniconda/bin:$PATH"
+        - export LC_ALL=en_US.UTF-8
+        - export LANG=en_US.UTF-8
+        - export AZURE_BLOB_KEY_QE72NMJYY6MBI="secret:azure-blob-key-qe72nmjyy6mbi"
```

Comment:
Is this still required?

Reply:
Yes,  tests in cli/test_copy.py/test_azure  and utils/test_store.py still  needs azure_store_cfg_factory, which needs env_name.
https://github.com/JDASoftwareGroup/kartothek/pull/307/files#diff-484462fced51d1a06b1d93b4a44dd535R169-R170

Reply:
It's not required, the tests will just be skipped. We need to clarify how to proceed with tests for our ABS-specific copy routines later, but let's do this AFTER this PR.